sources/platform/actors/development/quick_start/start_locally.md

```diff
@@ -1,89 +1,131 @@
 ---
 title: Local development
 sidebar_position: 1
-description: Create your first Actor locally on your machine and then deploy it to the Apify platform and run it in the cloud.
+description: Create your first Actor locally on your machine, deploy it to the Apify platform, and run it in the cloud.
 slug: /actors/development/quick-start/locally
 ---
 
-**Create your first Actor locally on your machine and then deploy it to the Apify platform and run it in the cloud.**
+**Create your first Actor locally on your machine, deploy it to the Apify platform, and run it in the cloud.**
 
 ---
 
-> Prerequisite - You need to have [Node.js](https://nodejs.org/en/) version 16 or higher with NPM installed on your computer.
+:::info Prerequisites
 
-## 1. Install Apify CLI
+You need to have [Node.js](https://nodejs.org/en/) version 16 or higher with NPM installed on your computer.
 
-On macOS (or Linux), you can install the Apify CLI via the Homebrew package manager.
+:::
+
+## Install Apify CLI
+
+### MacOS/Linux
+
+You can install the Apify CLI via the [Homebrew package manager](https://brew.sh/).
 
 ```bash
 brew install apify-cli
 ```
 
-Otherwise, use [NPM](https://www.npmjs.com/) to install the Apify CLI.
+### Other platforms
+
+Use [NPM](https://www.npmjs.com/) to install the Apify CLI.
 
 ```bash
 npm -g install apify-cli
 ```
 
-For more installation and advanced usage information, see the [Apify CLI documentation](https://docs.apify.com/cli/).
+Visit [Apify CLI documentation](https://docs.apify.com/cli/) for more information regarding installation and advanced usage.
 
-## 2. Create your Actor
+## Create your Actor
 
-You can use the following command to create an Actor:
+To create a new Actor, use the following command:
 
 ```bash
 apify create
 ```
 
-You will be prompted to name your Actor (e.g, my-actor), and choose the programming language you would like to use (JavaScript, TypeScript, or Python). Next, you will be presented with a list of Actor development templates. After choosing your preferred template the CLI will:
+The CLI will prompt you to:
 
-- Create a `my-actor` directory containing boilerplate code.
-- Install all the project dependencies.
+1. _Name your Actor_: Enter a descriptive name for your actor, such as `my-actor`
```

Comment:
> Enter a descriptive name for your actor, such as `my-actor`

The `my-actor` is not very descriptive 😄 maybe we can come with with something a bit better?